design/design-doc-stacks.md

```diff
@@ -290,7 +357,26 @@ The process of installing a Stack involves downloading its package, extracting t
 There are a few different options for this flow, which are described below, but a key aspect of the proposed implementation is to take advantage of the existing machinery in Kubernetes around container image download, verification, and extraction.
 This will be much more efficient and reliable than writing new code within Crossplane to do this work ourselves.
 
-Possible solutions for package processing are listed below, along with their pros and cons.
+The current approach for package processing abbreviated:
+
+* An `extension-request` object requires attention from the Extension Manager
```

Comment:
do you see a lot of value in this abbreviated description, or is the installation-flow linked just below good enough?

I'm wondering if we should just update the Installation Flow section with these details instead of having 2 sections addressing it.

Reply:
I added this to contrast the current implementation with some of the earlier design thoughts.

I'll revisit this, merging sections should be fine (if necessary).  I wrote what I had here before noticing the Installation Flow duplication on a subsequent pass.

Reply:
Updated to avoid duplication.